### ISSUE TITLE
User's content privacy settings

### ISSUE THREAD

**Issue by madzadev** [forem/forem#8291]
**Is your feature request related to a problem? Please describe.**

Currently all the articles and comments are being fully available to anyone in the web. I guess some users would classify some of the his/her information in DEV (especially comments) personal and probably would want to hide it from absolute strangers.

The initial idea of the feature comes from Twitter (see below).

![rtyry](https://user-images.githubusercontent.com/42511917/83859704-f937c900-a726-11ea-8740-fe66a5ed60e7.png)

**Describe the solution you'd like**

When a new user registers, the 'public' option is select by default. Same for all the already registered users. Here I made a quick sketch of how it could look like in user's settings:

![Untitled44](https://user-images.githubusercontent.com/42511917/83861359-29806700-a729-11ea-98f2-9e66882bfc8b.png)
**Describe alternatives you've considered**

Tho, in reality I couldn't imagine the forth case - articles as private and comments as public.
So, maybe, to eliminate that, it could look something like this:

![Untitleeeed](https://user-images.githubusercontent.com/42511917/83866162-0dcc8f00-a730-11ea-9434-dddb7a5639d5.png)

**Additional context**

Sorry for not using fonts and styling of DEV, it was not my intention.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Thanks for the request @madarsbiss 

This is tangentially related https://github.com/thepracticaldev/dev.to/issues/8212 though I think it's a slightly different request, so I'll keep both open for the time being :)

**Comment by mikeydorje** [comment]
Feature request from 2020 that never received product approval or `ready for dev` label. After 6 years with no activity, closing as the platform direction appears to have prioritized public content.

If privacy controls are still needed, please open a new issue with updated context or start a Discussion in the Features section.
